### PR TITLE
repin CWL version

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: src/ci/bin/test.inc.sh [hunk 1]
```diff
@@ -295,9 +294,6 @@ cromwell::private::checkout_pinned_cwl() {
         (
             cd "${CROMWELL_BUILD_CWL_TEST_DIRECTORY}" || exit 1
             git checkout "${CROMWELL_BUILD_CWL_TEST_COMMIT}"
-            # temporarily required to allow for use of the --timeout option to run_test.sh
-            # until we update our pinned conformance hash to include this commit on the checkout.
-            git cherry-pick -m 1 "${CROMWELL_BUILD_CWL_TEST_TIMEOUT_PATCH_COMMIT}"
```
Comment (mcovarr) [inline src/ci/bin/test.inc.sh]: 🎉

**Comment by mcovarr** [comment]
conformance 120 (and only 120) failed in both of your PAPI builds

**Comment by geoffjentry** [comment]
@mcovarr it was 💯 green before i pushed the PR so i'm gonna blame our very stable test suite for now

**Comment by geoffjentry** [comment]
@mcovarr to my point, restarting both resulted in 1 passing. Hooray for deterministic testing.

**Comment by geoffjentry** [comment]
@mcovarr looks like we're AOK. Your lack of faith is disturbing.

**Pull request merged**